openapi/parameters.yaml

```diff
@@ -225,3 +225,24 @@ user_id:
   in: path
   schema:
     type: string
+JournalBibstem:
+  name: bibstem
+  description: ADS identifier for publications embedded within bibcodes
+  in: path
+  required: true
+  schema:
+    type: string
+JournalVolume:
+  name: volume
+  description: publication volume
+  in: path
+  required: true
+  schema:
+    type: string
+JournalJournalName:
+  name: journalname
+  description: URL-encoded search string
```

Comment:
The `URL-encoded` part of this is tricky, because yes, the param needs to be URL-encoded; however, this doc site will auto-encode the value entered into the box (e.g. if you type `Astronomical Journal` into the box, then click Try, the curl that gets constructed is `https://api.adsabs.harvard.edu/v1/journals/Journal/Astronomical%20Journal` without you having to do anything special). I've been vague about this point elsewhere in the docs, so not a big deal, but I'm open to suggestions on this point

Reply:
I removed the URL-encoded qualifier to the text to be consistent with existing documentation.  But I think it might be good to specify somewhere in the documentation that in general, strings need to be URL-encoded.  My guess is that  anyone without any coding background that's trying curl on their own may be tripped up by this.